auth/token.go

```diff
@@ -147,5 +160,12 @@ func (t stdToken) GetAllClaimsAsMap() map[string]interface{} {
 }
 
 func (t stdToken) getJwtToken() jwt.Token {
+	// support ias custom domains
+	if t.IasIssuer() != "" {
+		err := t.jwtToken.Set("iss", t.IasIssuer())
```

Comment:
Why is the Issuer claim overwritten with the ias issuer in the parsed token?

Reply:
Hi @f-blass 
The `jwx` framework needs a std `Token` here https://github.com/SAP/cloud-security-client-go/blob/master/auth/validator.go#L82: 

```
jwt.Validate(t.getJwtToken(),...
```
Do you have a better idea?

Reply:
I see... Unfortunately there is no validation option which accepts a func. So we have to do it either that way or validate the issuer completely ourselfes after the validations from the framework. 
But as the `getJwtToken` is not public I guess we can use this workaround.